### PR TITLE
Lil sumptin

### PR DESCRIPTION
**Pull request by ThePrimeagen**
@polarmutex what do you think about this config for #11 ?

### CODE REVIEW

File: lua/harpoon/init.lua [hunk 1]
```diff
@@ -8,22 +8,32 @@ local cache_config = string.format("%s/harpoon.json", data_path)
 local M = {}
 
 --[[
+-- type Layout = "float" | "split" | "vsplit" | "hsplit" | "replace" (default) | "tab"
```
Comment (ThePrimeagen) [inline lua/harpoon/init.lua:11]: @polarmutex I am thinking that this would allow for every option.

Comment (polarmutex) [reply lua/harpoon/init.lua:11]: do we need spilt with hsplit and vsplit? or what is the difference between them?

**Review by polarmutex — COMMENTED** [review]
config looks good, just the  one question

File: README.md [hunk 2]
```diff
@@ -23,64 +23,6 @@ Plug 'ThePrimeagen/harpoon'
 ```
 
 ## Harpooning
+-- TODO -- Sorry (still being dev'd on)
 
-### The Basics
```
Comment (brandoncc) [inline README.md:27]: It seems like this readme is a bad idea to merge 😉

**Pull request closed without merging**